### ISSUE TITLE
Strange error when loading different notebooks

### ISSUE THREAD

**Issue by joelpires** [microsoft/praxxis#47]
When I try to open other notebooks besides the ones in the "test" folder, there's always an error like what it shows in the image.

You can open other random notebooks, or the error only happens to me?
![issue](https://user-images.githubusercontent.com/13687077/71759625-c2524400-2e64-11ea-84c6-b54df0ad4397.JPG)

**Comment by abertsch72** [comment]
That's interesting. Over the summer, we never ran into that -- but I cloned [the Python Data Science Handbook](https://github.com/jakevdp/PythonDataScienceHandbook) notebooks and ran into that error too. It seems to not occur on every notebook, either. Definitely a bug :/

**Comment by joelpires** [comment]
ok, nice!
Yeah, I would like to join the chat and give you a little hand. But I can't find the link :/
Can you post it here?

**Comment by abertsch72** [comment]
Yes, absolutely! The link is [here](https://riot.im/app/#/room/#praxxis:matrix.org)
